SU-SWS/ace-sdssgryphon
# SDSS-527: Updated sdss_subtheme decanter version and npm packages

# READY FOR REVIEW

# Summary
- Updated decanter and other npm packages for the sdss_subtheme

# Review By (Date)
- ASAP

# Urgency
- High

# Steps to Test

1. Checkout this branch
2. Navigate into the `sdss_subtheme` directory/folder and run the following 3 commands:
    - `nvm use`
    - `npm install` 
    - `npm run publish`
4. Run `git status` and confirm no files have changed

# Associated Issues and/or People
https://stanfordits.atlassian.net/browse/SDSS-527

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
